### ISSUE TITLE
WYSWIG can create empty links.

### ISSUE THREAD

**Issue by swirtSJW** [department-of-veterans-affairs/va.gov-cms#1801]
**Describe the defect**
If a link is edited, under some specific circumstances (which are not fully known) the WYSIWIG creates two links.   One with text, and one without.
 
![image](https://user-images.githubusercontent.com/5752113/82487646-ae724c00-9aac-11ea-8587-728fd5fd8231.png)

The empty one causes FE link tests to fail, thereby blocking content deployment.

Investigate whether CKeditor can be configured or has a solution to remove any links that only contain whitespace.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information if relevant, or delete):**

**Additional context**

## Labels
- [/] Issue type (red) (defaults to "Defect")
- [ ] CMS subsystem (green)
- [ ] CMS practice area (blue)
- [/] CMS objective (orange) (not needed for bug tickets)
- [ ] CMS-supported product (black)

**Comment by swirtSJW** [comment]
One option if we can not get WYSIWIG to automagically remove them would be to have the node link report call out links that have no text in the anchor.

**Comment by kevwalsh** [comment]
See also #4806

**Comment by laflannery** [comment]
Closing this as a duplicate of [#4806](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/4806)
